babycenter/static/css/modules/_article.scss

```diff
@@ -11,7 +11,12 @@
 }
 
 .article__date {
-  color: #999;
+  color: #ccc;
+}
+
+.section-article_date {
```

Comment:
perhaps we should rename `section-article_date` to something like `article__date--italic`

I don't know if italic is a good name though?